models/clm/CAM_bias_correction/CAM4_site_grid.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Extracts site level grid cell from CAM4 reanalysis
+
+# CAM4 reanalysis
+CAM4_grid_path="/glade/collections/rda/data/ds199.1/"
+# Site Level grid cell from CAM4 renanalysis
+CAM4_site_path="/glade/work/bmraczka/CAM4_NR1/"
+
+# The US-NR1 flux tower site is used as an example 
+# NR1 location (40.03, -105.55) or (40.03, 254.45)
+# CAM4 grid is 1.95x2.5  or 288 longitude grids and 96 latitude grids
```

Comment:
Should be "CAM4 grid is 1.95x2.5 degrees, or 96 latitude and 144 longitude points"

Reply:
grids -> grid points in 351a324e4eaf9c40c958889624097f491c9f4b44

Leaving the 'used for other profiles for a future pull reqeust.